e2e_testing/torchscript/basic.py

```diff
@@ -738,3 +738,51 @@ def forward(self, input, tensor1, tensor2):
 @register_test_case(module_factory=lambda: AddCDivModule())
 def AddCDivModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(1,3), tu.rand(1,3), tu.rand(1,3))
+
+class Fill_TensorF64WithF32(torch.nn.Module):
```

Comment:
nit: F64, F32, I64 are llvm type conventions. Prefer to use Float64, Float32, Int64

Reply:
Done.